src/lib/components/lines-and-dots/event-type-filter.svelte

```diff
@@ -159,6 +162,7 @@
           checked={$eventTypeFilter.some((type) => type === option.value)}
           label={option.label}
           labelHidden
+          class="mt-px"
```

Comment:
What do these mt-px classes do? I'm confused maybe set mt to 0? If that's the case maybe we do mt-0 instead?